<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>drawPicXmlClass</class>
 <widget class="QMainWindow" name="drawPicXmlClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>drawPicXml</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>715</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenFiles"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionSourse_code"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuListSize">
     <property name="title">
      <string>ListSize</string>
     </property>
     <addaction name="actionIconModel"/>
     <addaction name="actionListModel"/>
     <addaction name="actionOriginal"/>
    </widget>
    <addaction name="actionGenerate_XML"/>
    <addaction name="actionProcess_pictures"/>
    <addaction name="actionShow_Dock"/>
    <addaction name="menuListSize"/>
    <addaction name="actionClearBindingBox"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dock">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="windowTitle">
    <string>Properities</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelFolder">
         <property name="text">
          <string>folder:</string>
         </property>
         <property name="buddy">
          <cstring>comboBoxFolder</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxFolder">
         <item>
          <property name="text">
           <string>VOC2007</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VOC2008</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VOC2009</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VOC2010</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VOC2011</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VOC2012</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labelFileName">
         <property name="text">
          <string>fileName:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelFileName_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="labelDataBase">
         <property name="text">
          <string>dataBase:</string>
         </property>
         <property name="buddy">
          <cstring>comboBoxDataBase</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxDataBase">
         <item>
          <property name="text">
           <string>The VOC2007 Database</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>The VOC2008 Database</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>The VOC2009 Database</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>The VOC2010 Database</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>The VOC2011 Database</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>The VOC2012 Database</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="labelFlickrid">
         <property name="text">
          <string>flickrid:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelFlickrid_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="labelOwnerName">
         <property name="text">
          <string>ownerName:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelOwnerName_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="labelDepth">
         <property name="text">
          <string>depth:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelDepth_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="labelWidth">
         <property name="text">
          <string>width:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelWidth_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="labelHeight">
         <property name="text">
          <string>height:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelHeight_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>240</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionSourse_code">
   <property name="text">
    <string>Sourse Code</string>
   </property>
  </action>
  <action name="actionGenerate_XML">
   <property name="text">
    <string>Generate XML</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionProcess_pictures">
   <property name="text">
    <string>Process pictures</string>
   </property>
  </action>
  <action name="actionShow_Dock">
   <property name="text">
    <string>Dock</string>
   </property>
  </action>
  <action name="actionOpenFiles">
   <property name="text">
    <string>OpenFiles</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionChangeListSize">
   <property name="text">
    <string>ChangeListSize</string>
   </property>
  </action>
  <action name="actionIconModel">
   <property name="text">
    <string>IconModel</string>
   </property>
  </action>
  <action name="actionListModel">
   <property name="text">
    <string>ListModel</string>
   </property>
  </action>
  <action name="actionOriginal">
   <property name="text">
    <string>Original</string>
   </property>
  </action>
  <action name="actionClearBindingBox">
   <property name="text">
    <string>ClearBindingBox</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
